packages/app/src/specs/banners/CreateOrganizationBanner.cy.tsx

```diff
@@ -0,0 +1,21 @@
+import CreateOrganizationBanner from './CreateOrganizationBanner.vue'
+
+describe('<CreateOrganizationBanner />', () => {
+  it('should render expected content', () => {
+    const linkHref = 'http://dummy.cypress.io/organizations/create'
+
+    cy.gqlStub.Query.cloudViewer = {
+      __typename: 'CloudUser',
+      id: 'test123',
+      cloudOrganizationsUrl: linkHref,
+    } as any
+
+    cy.mount({ render: () => <CreateOrganizationBanner modelValue={true} /> })
+
+    cy.get('a')
+    .should('have.attr', 'href')
+    .and('contain', linkHref)
```

Comment:
Should we assert the `specPage.banners.Join or create an organization in Cypress Dashboard to access your projects and recorded test runs` here? Since grabbing that content is also a responsibility of this component.